### PR TITLE
[Github actions] Properly use v1.0.2 for all actions

### PR DESCRIPTION
**Pull request by SerhatG**

### CODE REVIEW

**Review by BertScholten — APPROVED** [review]
LGTM

File: .github/workflows/on-pull_request-opened-synchronize-reopened.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
           path: aerius-github-actions
           ref: v1
```
Comment (BertScholten) [inline .github/workflows/on-pull_request-opened-synchronize-reopened.yml:17]: Shouldn't this be v1.0.2 as well?

Comment (SerhatG) [reply .github/workflows/on-pull_request-opened-synchronize-reopened.yml:17]: Ah yes, apparently this was never updated to 1.0.2 like the rest in a previous PR.

**Review by BertScholten — APPROVED** [review]
LGTM

**Pull request merged**